builds/jazz-build-module/whitelist-validator-module.groovy

```diff
@@ -0,0 +1,139 @@
+        - DescribeBackup : "*"
+        - DescribeContinuousBackups : "*"
+        - DescribeGlobalTable : "*"
+        - DescribeGlobalTableSettings : "*"
+        - DescribeLimits : "*"
+        - DescribeReservedCapacity : "*"
+        - DescribeReservedCapacityOfferings : "*"
+        - DescribeStream : "*"
+        - DescribeTable : "*"
+        - DescribeTimeToLive : "*"
+        - EnclosingOperation : "*"
+        - GetItem : "*"
+        - GetRecords : "*"
+        - GetShardIterator : "*"
+        - ListBackups : "*"
+        - ListGlobalTables : "*"
+        - ListStreams : "*"
+        - ListTables : "*"
+        - ListTagsOfResource : "*"
+        - PurchaseReservedCapacityOfferings : "*"
+        - PutItem : "*"
+        - Query : "*"
+        - RestoreTableFromBackup : "*"
+        - RestoreTableToPointInTime : "*"
+        - Scan : "*"
+        - TagResource : "*"
+        - UntagResource : "*"
+        - UpdateContinuousBackups : "*"
+        - UpdateGlobalTable : "*"
+        - UpdateGlobalTableSettings : "*"
+        - UpdateItem : "*"
+        - UpdateTable : "*"
+        - UpdateTimeToLive : "*"
+  ApiGateway:
+    - Type: "AWS::ApiGateway::Account"
+    - Type: "AWS::ApiGateway::ApiKey"
+    - Type: "AWS::ApiGateway::Authorizer"
+    - Type: "AWS::ApiGateway::BasePathMapping"
+    - Type: "AWS::ApiGateway::ClientCertificate"
+    - Type: "AWS::ApiGateway::Deployment"
+    - Type: "AWS::ApiGateway::DocumentationPart"
+    - Type: "AWS::ApiGateway::DocumentationVersion"
+    - Type: "AWS::ApiGateway::DomainName"
+    - Type: "AWS::ApiGateway::GatewayResponse"
+    - Type: "AWS::ApiGateway::Method"
+    - Type: "AWS::ApiGateway::Model"
+    - Type: "AWS::ApiGateway::RequestValidator"
+    - Type: "AWS::ApiGateway::Resource"
+    - Type: "AWS::ApiGateway::RestApi"
+    - Type: "AWS::ApiGateway::Stage"
+    - Type: "AWS::ApiGateway::UsagePlan"
+    - Type: "AWS::ApiGateway::UsagePlanKey"
+    - Type: "AWS::ApiGateway::VpcLink"
+  LambdaFunction:
+    - Type: "AWS::Lambda::EventSourceMapping"
+    - Type: "AWS::Lambda::Function"
+    - Type: "AWS::Lambda::Alias"
+    - Type: "AWS::Lambda::LayerVersion"
+    - Type: "AWS::Lambda::LayerVersionPermission"
+    - Type: "AWS::Lambda::Permission"
+    - Type: "AWS::Lambda::Version"
+  S3:
+    - Type: "AWS::S3::Bucket"
+    - Type: "AWS::S3::BucketPolicy"
+  CloudFront:
+    - Type: "AWS::CloudFront::Distribution"
+    - Type: "AWS::CloudFront::CloudFrontOriginAccessIdentity"
+    - Type: "AWS::CloudFront::StreamingDistribution"
+  SQS:
+    - Type: "AWS::SQS::Queue"
+    - Type: "AWS::SQS::QueuePolicy"
+  Kinesis:
+    - Type: "AWS::Kinesis::Stream"
+    - Type: "AWS::Kinesis::StreamConsumer"
+    - Type: "AWS::KinesisFirehose::DeliveryStream"
+    - Type: "AWS::KinesisAnalytics::Application"
+    - Type: "AWS::KinesisAnalytics::ApplicationOutput"
+    - Type: "AWS::KinesisAnalytics::ApplicationReferenceDataSource"
+  IAM:
+    - Type: "AWS::IAM::AccessKey"
```

Comment:
Are actions missing here (& other resource types)?

Reply:
Now the whitelist.yml also contains 'actions'

Reply:
However they are not used just yet

Reply:
Resolved with https://github.com/tmobile/jazz/pull/829/commits/70374d203a7cb8338de2d84ee1c4902831a57b35